src/test/java/ConverterTest.java

```diff
@@ -0,0 +1,537 @@
+import converters.*;
+import exceptions.ConverterException;
+import options.IOption;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static options.ArrayOption.*;
+import static options.BitsOption.LEFT;
+import static options.BitsOption.RIGHT;
+import static options.IntOption.BIG;
+import static options.IntOption.LITTLE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class ConverterTest {
+	BytesConverter byteConverter = new BytesConverter();
+	HexConverter hexConverter = new HexConverter();
+	IntConverter intConverter = new IntConverter();
+	BitsConverter bitsConverter = new BitsConverter();
+	ArrayConverter arrayConverter = new ArrayConverter();
+
+	@ParameterizedTest
+	@ValueSource(strings = {"test", "12t", "-12"})
+	void testFromIntInvalid(String input) {
+		assertThrows(ConverterException.class, () -> intConverter.convertFrom(input, null));
+	}
+
+	@ParameterizedTest
+	@ValueSource(strings = {"12G", "test", "35FH"})
+	void testFromHexInvalid(String input) {
+		assertThrows(ConverterException.class, () -> hexConverter.convertFrom(input, null));
+	}
+
+	@ParameterizedTest
+	@ValueSource(strings = {"012", "test"})
+	void testFromBitsInvalid(String input) {
+		assertThrows(ConverterException.class, () -> bitsConverter.convertFrom(input, null));
+	}
+
+	@Test
+	void testBytesToBytes() {
+		try {
+
+			String bits = byteConverter.convertFrom("test");
```

Comment:
**issue:** not DRY (don't repeat yourself) tests

I recommend to make helper class for common code that you reuse in every single test.